packages/user-operation-controller/src/types.ts

```diff
@@ -253,6 +253,15 @@ export type UpdateUserOperationResponse = {
    * Not required if a paymaster is not sponsoring the transaction.
    */
   paymasterAndData?: string;
+
+  /**
+   * The final gas limits for the user operation suggested by the smart contract account.
+   * The simulated gas limits may be different after the bundler estimates gas with the use
+   * of the paymaster.
+   */
+  callGasLimit?: string;
+  preVerificationGas?: string;
+  verificationGasLimit?: string;
```

Comment:
This type is already defined in the API [here](https://github.com/MetaMask/keyring-api/blob/main/src/eth/erc4337/types.ts#L67-L72), we should probably reuse it.